core/src/toga/widgets/base.py

```diff
@@ -17,10 +19,29 @@
 StyleT = TypeVar("StyleT", bound=BaseStyle)
 
 
+# based on colors from https://davidmathlogic.com/colorblind
+debug_background_palette = [
+    "#d0e2ed",  # very light blue
+    "#b8d2e9",  # light blue
+    "#f8ccb0",  # light orange
+    "#f6d3be",  # soft orange
+    "#c7e7b2",  # light green
+    "#f0b2d6",  # light pink
+    "#e5dab0",  # light yellow
+    "#d5c2ea",  # light lavender
+    "#b2e4e5",  # light teal
+    "#e5e4af",  # light cream
+    "#bde2dc",  # soft turquoise
+]
+shuffle(debug_background_palette)
```

Comment:
Why programmatically shuffle? We definitely need an order which cycles good contrast (i.e., we don't have "light blue" next to "very light blue"), but that order doesn't need to be random.

Plus, randomisation takes more time at startup, and a randomised order could be jarring for as successive runs won't give the same output.